internal/repository/migrations/000002_create_nodestate_epochs_claims_proofs.up.sql

```diff
@@ -0,0 +1,47 @@
+-- (c) Cartesi and individual authors (see AUTHORS)
+-- SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+CREATE TABLE "epochs"
```

Comment:
I don't know, my gut feeling is always to not have this kind of id values that are just counting from 0 and have no real purpose. It can make things easier, but it can also be another value we have to consider and maintain through all levels of the architecture, including models, the readerclient, etc

But honestly, I'm not sure. We should bring this topic to the team and discuss it further.